models/users.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection.js");
+
+exports.checkUserExists = (username) => {
```

Comment:
Though having a checkUserExists function is a valid way to approach the problem of the user not existing- a PSQL error happens when you try and submit a comment with an invalid username. It would be a less code-heavy response to just add another block to your PSQL error catching. You could refactor this - though it's not mandatory